File: CPP/Program/Product.cpp
/* Saya Rifa Sania NIM 2206697 mengerjakan Latihan Praktikum 2 dalam mata kuliah Desain Pemrograman Berorientasi Objek
untuk keberkahanNya maka saya tidak melakukan kecurangan seperti yang telah dispesifikasikan. Aamiin... */

// include library
#include <iostream>
#include <string>

using namespace std;

class Product{
    // atribut
    private:
        string idProduct;
        string name;
        string brand;
        string price;

    public:
        Product(){
            // konstruktor kosong
        }

        // set and get

        void setIdProduct(string idProduct){
            this->idProduct = idProduct;
        }
        
        string getIdProduct(){
            return this->idProduct;
        }

        void setName(string name){
            this->name = name;
        }

        string getName(){
            return this->name;
        }

        void setBrand(string brand){
            this->brand = brand;
        }

        string getBrand(){
            return this->brand;
        }

        void setPrice(string price){
            this->price = price;
        }

        string getPrice(){
            return this->price;
        }
};

File: PYTHON/Program/Product.py
class Product:

    __idProduct = ""
    __name = ""
    __brand = ""
    __price = ""

    def __init__(self):
        self.__idProduct = ""
        self.__name = ""
        self.__brand = ""
        self.__price = ""

    def setIdProduct(self, idProduct):
        self.__idProduct = idProduct

    def getIdProduct(self):
        return self.__idProduct
    
    def setName(self, name):
        self.__name = name

    def getName(self):
        return self.__name
    
    def setBrand(self, brand):
        self.__brand = brand

    def getBrand(self):
        return self.__brand
    
    def setPrice(self, price):
        self.__price = price

    def getPrice(self):
        return self.__price